06:19:31 PM:DEBUG[logger_base.py:12] Mensaje a nivel debug
06:19:31 PM:INFO[logger_base.py:13] Mensaje a nivel info
06:19:31 PM:WARNING[logger_base.py:14] Mensaje a nivel warning
06:19:31 PM:ERROR[logger_base.py:15] Mensaje a nivel error
06:19:31 PM:CRITICAL[logger_base.py:16] Mensaje a nivel critical
11:30:02 AM:DEBUG[Persona.py:48] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
            Email: jperez@mail.com
        
11:38:46 AM:DEBUG[Persona.py:48] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
            Email: jperez@mail.com
        
11:39:30 AM:DEBUG[Persona.py:48] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
            Email: jperez@mail.com
        
11:39:30 AM:DEBUG[Persona.py:50] 
            Id Persona: None,
            Nombre: Jose,
            Apellido: Lepez,
            Email: ljose@mail.com
        
11:39:56 AM:DEBUG[Persona.py:48] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
            Email: jperez@mail.com
        
11:39:56 AM:DEBUG[Persona.py:50] 
            Id Persona: None,
            Nombre: Jose,
            Apellido: Lepez,
            Email: ljose@mail.com
        
11:39:56 AM:DEBUG[Persona.py:52] 
            Id Persona: 1,
            Nombre: None,
            Apellido: None,
            Email: None
        
12:35:25 PM:ERROR[conexion.py:26] Ocurrió un errorconnection to server at "127.0.0.1", port 5432 failed: FATAL:  database "test_db" does not exist

12:35:54 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7ff225ff5540; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
12:42:39 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7fa369661540; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
12:43:19 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f50e5094540; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
12:43:19 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f50e5092a90; closed: 0>
02:23:09 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f3b084d1680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
02:23:09 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f3b0742e400; closed: 0>
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 1,
            Nombre: Juan Carlos,
            Apellido: Roldan,
            Email: jcroldan@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 2,
            Nombre: Carla,
            Apellido: Gomez
,
            Email: kgomez@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 9,
            Nombre: Carlos,
            Apellido: Lara,
            Email: clara@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 10,
            Nombre: Marcos,
            Apellido: Canto,
            Email: mcanto@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 11,
            Nombre: Juan,
            Apellido: Juarez,
            Email: jcjuarez@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 13,
            Nombre: Maria,
            Apellido: Esparza,
            Email: mesparza@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 19,
            Nombre: Carlos,
            Apellido: Lara,
            Email: clara@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 20,
            Nombre: Carlos,
            Apellido: Lara,
            Email: clara@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 21,
            Nombre: Braian,
            Apellido: Troncoso,
            Email: bTroncoso@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 22,
            Nombre: Juan Carlos,
            Apellido: Perez,
            Email: jPerez@mail.com
        
02:23:09 PM:DEBUG[persona_dao.py:39] 
            Id Persona: 23,
            Nombre: Alex,
            Apellido: Rojas,
            Email: arojas@mail.com
        
02:48:31 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f57dfda9680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
02:48:31 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f57ded04400; closed: 0>
02:50:11 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f5642ffe680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
02:50:11 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f5641f59400; closed: 0>
02:51:13 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f3734591680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
02:51:13 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f37334ec400; closed: 0>
02:53:26 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f7d663de680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
02:53:26 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f7d65339400; closed: 0>
02:53:26 PM:DEBUG[persona_dao.py:42] Persona insertada: 
            Id Persona: None,
            Nombre: Pedro,
            Apellido: Romero,
            Email: promero@mail.com
        
02:53:26 PM:DEBUG[persona_dao.py:51] Personas insertadas: 1
03:07:16 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f66f73f9680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:07:16 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f66f6354400; closed: 0>
03:07:16 PM:DEBUG[persona_dao.py:41] Persona insertada: 
            Id Persona: None,
            Nombre: Omero,
            Apellido: Ramos,
            Email: omeror@mail.com
        
03:07:16 PM:DEBUG[persona_dao.py:50] Personas insertadas: 1
03:22:20 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7fd1e94e6680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:22:20 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7fd1e8441400; closed: 0>
03:26:04 PM:DEBUG[conexion.py:23] Conexión Exitosa: <connection object at 0x7f53b0146680; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:26:04 PM:DEBUG[conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x7f53af0a1400; closed: 0>
03:26:04 PM:DEBUG[persona_dao.py:52] Persona actualizada: 
            Id Persona: 1,
            Nombre: Juan Jose,
            Apellido: Pena,
            Email: jjpena@mail.com
        
03:26:04 PM:DEBUG[persona_dao.py:59] Personas actualizadas: 1
